.github/workflows/go.yaml

```diff
@@ -87,21 +93,15 @@ jobs:
       - name: Test with TinyGo
         # This only tests the apex package, not ./internal/build
         # due to TinyGo support for net.UDPConn and net.IPConn
-        run: tinygo test -bench . ./
+        run: tinygo test -bench -target=wasip1 . ./
```

Comment:
this change was required because of a bug that will be addressed in version 0.37.1

```
cannot resolve packages: GOARCH=wasm but GOOS is unset. Please set GOOS to wasm, wasip1, or wasip2.
```